### DISCUSSION TITLE
About difference between Enterprise and Business features

### DISCUSSION THREAD

**Discussion by adminclover** [community/community#123055] [Enterprise]
### Select Topic Area

Question

### Body

I am currently using business account , so I just want to know what are the new features are provided by enterprise and what are benefits of those extra features. Kindly give with example.

**Comment by LiteBrite82** [comment]
Hi @adminclover 

I'm including a link to our [account documentation that covers the basics](https://docs.github.com/en/get-started/learning-about-github/types-of-github-accounts). I'm also going to move this discussion over to the `Enterprise` category to give your question more exposure to Enterprise users. 

Thank you for joining our community discussions!!!
